packages/story-editor/src/utils/generateBlurhash.worker.js

```diff
@@ -16,7 +16,9 @@
 /**
  * External dependencies
  */
-import { encode } from 'blurhash';
+// Disable reason: loading in blurhash from from skypack.
+// eslint-disable-next-line import/no-unresolved
+import { encode } from 'https://cdn.skypack.dev/blurhash@1.1.4';
```

Comment:
🤔 Loading the webworker inline works but then webpack couldn't resolve `import x from 'blurhash'` to anything. This solution works, but let me know if y'all would prefer to do it another way.

Was having trouble getting webpack to map `from 'blurhash'` to a package.

Reply:
Definitely need another way here, not loading from a CDN.

In an inlined worker, wouldn‘t blurhash be inlined here?

This should ideally still work the same as on main branch

Reply:
Working with it last week - `blurhash` was not inlined. Will check into that today. Not sure why it wasn't inlined - but seemed like webpack was not able to resolve `import { encode } from 'blurhash'`.

Reply:
Specifically the error I get is:
```js
TypeError: Failed to resolve module specifier 'blurhash'
```

Reply:
That sounds like an error trying to import an ES6 module, probably because it was not inlined by webpack and maybe also related to that `type: module` change which seems unneeded.

Reply:
I think you're right!